src/main/java/server/Server.java

```diff
@@ -3,13 +3,15 @@
 import java.io.IOException;
 import java.net.ServerSocket;
 import java.net.Socket;
+import java.util.*;
 
 import messaging.Message;
 import socket.SocketManager;
 
 public class Server implements Runnable{
 	private int port;
 	private ServerSocket servSoc;
+	public static List<ClientConnection> clientList = new ArrayList<ClientConnection>();
```

Comment:
To avoid having a static datastructure, we can make ClientManager a property class of Server.